src/installer/tests/HostActivation.Tests/DependencyResolution/ResolveComponentDependencies.cs

```diff
@@ -449,10 +454,10 @@ public TestApp CreateComponentWithDependencies(Action<NetCoreAppBuilder> customi
                 NetCoreAppBuilder builder = NetCoreAppBuilder.PortableForNETCoreApp(componentWithDependencies)
                     .WithProject(p => p.WithAssemblyGroup(null, g => g.WithMainAssembly()))
                     .WithProject("ComponentDependency", "1.0.0", p => p.WithAssemblyGroup(null, g => g.WithAsset("ComponentDependency.dll")))
-                    .WithPackage("Newtonsoft.Json", "13.0.1", p => p.WithAssemblyGroup(null, g => g
-                        .WithAsset("lib/netstandard1.0/Newtonsoft.Json.dll", f => f
-                            .WithVersion("13.0.0.0", "13.0.1.25517")
-                            .WithFileOnDiskPath("Newtonsoft.Json.dll"))))
+                    .WithPackage(AdditionalDependencyName, "2.0.1", p => p.WithAssemblyGroup(null, g => g
+                        .WithAsset($"lib/netstandard1.0/{AdditionalDependencyName}.dll", f => f
+                            .WithVersion("2.0.0.0", "2.0.1.23344")
```

Comment:
If this is dummy file, where do we set 2.0.1.23344?

Reply:
It's all made up - even the version numbers. I intentionally made up something which looks like a file version (the first version is assembly version, the second is file version).
It's written (by the test) into a .deps.json which is then parsed. Currently the test is probably not even using the version for anything, but it makes a more realistic test (since SDK will write the versions there always).